### PR TITLE
test(dp itx): for mongodb make test.failing

### PR DESCRIPTION
**Pull request by Jolg42**
Context: https://prisma-company.slack.com/archives/C04F60M3G06/p1694527695727029

### CODE REVIEW

File: dataproxy/nodejs-mongodb-itx/test/long-running.test.ts [hunk 1]
```diff
@@ -36,7 +36,11 @@ describe('long-running', () => {
     })
   })
 
-  testIf(!isDP1)(
+  // While testing we figure out that MongoDB is using DP1
+  // So we made the test a `test.failing` test
+  // TODO: Remove this once we have DP2/accelerate implementation for MongoDB
```
Comment (janpio) [inline dataproxy/nodejs-mongodb-itx/test/long-running.test.ts:41]: We do! It's just not being properly tested, but https://github.com/prisma/ecosystem-tests/pull/3939 does that (by adding the necessary env vars and changing the code to use them)

Comment (janpio) [reply dataproxy/nodejs-mongodb-itx/test/long-running.test.ts:41]:
```suggestion
  // TODO: Remove this when https://github.com/prisma/ecosystem-tests/pull/3939 adds testing for DP2/Accelerate
```

**Review by janpio — APPROVED** [review]

File: dataproxy/nodejs-mongodb-itx/test/long-running.test.ts (hunk 1 above)
Comment (Jolg42) [reply dataproxy/nodejs-mongodb-itx/test/long-running.test.ts:41]: Oh right ...!

Comment (Jolg42) [reply dataproxy/nodejs-mongodb-itx/test/long-running.test.ts:41]: Let's clean that up because it's very confusing

**Pull request merged**